src/test/ui/panics/panic-short-backtrace-windows-x86_64.rs

```diff
@@ -1,3 +1,7 @@
+// This test has been spuriously failing a lot recently (#92000).
+// Ignore it until the underlying issue is fixed.
+// ignore-msvc
```

Comment:
Is there a way to ignore the test on all platforms? I'm pretty sure it's only run on msvc so this should fully ignore it, but it'd be nice to be certain.

Reply:
It's ran on `windows-msvc` and `windows-gnu` but seems to be some kind of MSVC bug because it doesn't fail anywhere  else.

Reply:
Below it says

```
// only-x86_64-pc-windows-msvc
```

so I think it's not even run on `windows-gnu`.

Reply:
I believe you can do `// ignore-test` which is used in a bunch of other tests already.